config/src/config/mod.rs

```diff
@@ -325,21 +325,18 @@ impl NodeConfig {
         self.test = Some(test);
     }
 
-    #[cfg(any(test, feature = "fuzzing"))]
```

Comment:
Can we remove these all?! :D

Reply:
I'm now leaning toward the opinion that excessive feature exclusion is not really worth the headache....